stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -148,21 +122,44 @@ extension ${Self}: LosslessStringConvertible {
   ///       // p?.isNaN == true
   ///       // String(p!) == "nan(0x10)"
   ///
-  /// - A string in any other format or containing additional characters
-  ///   results in a `nil` value. For example, the following conversions
-  ///   result in `nil`:
+  /// A string in any other format than those described above
+  /// or containing additional characters
+  /// results in a `nil` value. For example, the following conversions
+  /// result in `nil`:
   ///
   ///       ${Self}(" 5.0")      // Includes whitespace
   ///       ${Self}("±2.0")      // Invalid character
   ///       ${Self}("0x1.25e4")  // Incorrect exponent format
   ///
-  /// - Parameter text: An input string to convert to a `${Self}?` instance.
+  /// A decimal or hexadecimal string is converted to a `${Self}`
+  /// instance using the IEEE 754 roundTiesToEven (default) rounding
+  /// attribute.
+  /// Values with absolute value smaller than `${Self}.leastNonzeroMagnitude`
+  /// are rounded to plus or minus zero.
+  /// Values with absolute value larger than `${Self}.greatestFiniteMagnitude`
+  /// are rounded to plus or minus infinity.
   ///
-  /// Backwards compatibility note:  Prior to Swift 5.4, a decimal or
+  ///       let y = ${Self}("1.23e-9999")
+  ///       // y == 0.0
+  ///       // y?.sign == .plus
+  ///
+  ///       let z = ${Self}("-7.89e-7206")
+  ///       // z == -0.0
+  ///       // z?.sign == .minus
+  ///
+  ///       let r = ${Self}("1.23e17802")
+  ///       // r == ${Self}.infinity
+  ///
+  ///       let s = ${Self}("-7.89e7206")
+  ///       // s == ${Self}.-infinity
+  ///
+  /// Note:  Prior to Swift 5.4, a decimal or
```

Comment:
I suggested spelling this `- Note:` to create an aside box like you see here:

https://developer.apple.com/documentation/swift/int32/2884321

Which comes from here in source:

https://github.com/apple/swift/blob/main/stdlib/public/core/IntegerTypes.swift.gyb#L94